packages/commonwealth/server.ts

```diff
@@ -54,8 +54,7 @@ const start = async () => {
     withLoggingMiddleware: true,
     withStatsMiddleware: true,
     withFrontendBuild: !NO_CLIENT,
-    withPrerender:
-      PRODUCTION && !NO_PRERENDER && SERVER_URL.includes('commonwealth.im'),
+    withPrerender: !!PRERENDER_TOKEN
```

Comment:
what happens here?

Reply:
@masvelio I am taking on this PR. I don't have the full context here, are we talking about whether we want to update this prop or not?

Reply:
I am just not sure why this change has been made?

Reply:
ok it looks like this was done to enable pre-renderer on all environments, per the removal of `SERVER_URL.includes('commonwealth.im')`, still unsure why its done tho. I will be reverting this if you guys don't have a strong argument against keeping it.

cc: @masvelio @burtonator

Reply:
well, my opinion is that if requirements are met without this change and everything works as expected, we should revert it. My guess is that is was changed to be able to test it on frick/frack and the intention should be to use prerender only on production

Reply:
reverted. Thanks for the feedback